splunk_eventgen/lib/eventgentoken.py

```diff
@@ -350,7 +350,7 @@ def _getReplacement(self, old=None, earliestTime=None, latestTime=None, s=None,
                 except:
                     logger.error("Could not parse json for '%s' in sample '%s'" % (listMatch.group(1), s.name))
                     return old
-                return random.choice(value)
+                return random.SystemRandom().choice(value)
```

Comment:
Use `SystemRandom` class which uses the system function `os.urandom()` to generate random numbers from sources provided by the operating system.